### ISSUE TITLE
Global all labels ending in "-OR" --> change to "-ION" ending

### ISSUE THREAD

**Issue by strambc** [OmegaProject/Omega#146]
This is a general comment:

all suffixes in "-or" have to be globally changed to "-ion":

"Estimator" --> "Estimation"
"Detector" --> "Detection"
"Editor" --> "Editing"

**Comment by Akramhar** [comment]
This issue is way to general. I'll close it for the moment.
If you notice something that needs to be changed I need more precise coordinate of where to find it.

**Comment by strambc** [comment]
Can there be a global change?
I listed the labels i want changed

**Comment by Akramhar** [comment]
"Global" change is something I would like to avoid as much as possible.
Just to give you an example:
I went through #99 and I renamed all plugin in the way you specified on the file.
Then I reach this ticket and I should change the "Mosaic 2D particle detector" plugin name in "Mosaic 2D particle detection".
Is this meant to be changed? How can I be sure?
Another example on the same line the plugin "OMEGA Tracking Editor" that would get changed in "OMEGA Tracking Editing".
Hope this clarifies what I mean.

**Comment by strambc** [comment]
I understand your frustration.
However as we are getting closer to the publication it is typical for things to be changed back and forth a number of times.
I remember being in your position and getting very frustrated, angry even, with my supervisor about things like this…
However I now understand that it is not a static thing.
It is a moving target.
For example the structure of the paper, the order of the figures was changed at least 10 times if not more.
The idea is to aim for the best possible solution asymptotically…


On May 2, 2017, at 10:19, Akramhar <notifications@github.com<mailto:notifications@github.com>> wrote:


"Global" change is something I would like to avoid as much as possible.
Just to give you an example:
I went through #99<https://github.com/OmegaProject/Omega/issues/99> and I renamed all plugin in the way you specified on the file.
Then I reach this ticket and I should change the "Mosaic 2D particle detector" plugin name in "Mosaic 2D particle detection".
Is this meant to be changed? How can I be sure?
Another example on the same line the plugin "OMEGA Tracking Editor" that would get changed in "OMEGA Tracking Editing".
Hope this clarifies what I mean.

**Comment by strambc** [comment]
I will tell you what.
I am planning to go over the Algorithm descriptions again today or tomorrow and go over all results tables and labels.
Things will definitely be changed from where they are now.
I will review everything again and then decide what changes are needed and which are not.

Sounds good?


On May 2, 2017, at 11:49, Strambio, Caterina <Caterina.Strambio@umassmed.edu<mailto:Caterina.Strambio@umassmed.edu>> wrote:

I understand your frustration.
However as we are getting closer to the publication it is typical for things to be changed back and forth a number of times.
I remember being in your position and getting very frustrated, angry even, with my supervisor about things like this…
However I now understand that it is not a static thing.
It is a moving target.
For example the structure of the paper, the order of the figures was changed at least 10 times if not more.
The idea is to aim for the best possible solution asymptotically…


On May 2, 2017, at 10:19, Akramhar <notifications@github.com<mailto:notifications@github.com>> wrote:


"Global" change is something I would like to avoid as much as possible.
Just to give you an example:
I went through #99<https://github.com/OmegaProject/Omega/issues/99> and I renamed all plugin in the way you specified on the file.
Then I reach this ticket and I should change the "Mosaic 2D particle detector" plugin name in "Mosaic 2D particle detection".
Is this meant to be changed? How can I be sure?
Another example on the same line the plugin "OMEGA Tracking Editor" that would get changed in "OMEGA Tracking Editing".
Hope this clarifies what I mean.
